expect/expect.ts

```diff
@@ -212,22 +167,4 @@ export function expect(value: unknown, customMessage?: string): Expected {
   return self;
 }
 
-// a helper type to match any function. Used so that we only convert functions
-// to return a promise and not properties.
-// deno-lint-ignore no-explicit-any
-type Fn = (...args: any[]) => unknown;
-
-// converts all the methods in an interface to be async functions
-export type Async<T> = {
-  [K in keyof T]: T[K] extends Fn
-    ? (...args: Parameters<T[K]>) => Promise<ReturnType<T[K]>>
-    : T[K];
-};
-
-function isPromiseLike(value: unknown): value is PromiseLike<unknown> {
-  if (value == null) {
-    return false;
-  } else {
-    return typeof ((value as Record<string, unknown>).then) === "function";
-  }
-}
+expect.addEqualityTester = addCustomEqualityTester;
```

Comment:
Looks like this needs to be `addEqualityTesters` (plural `s`) https://jestjs.io/docs/expect#expectaddequalitytesterstesters

Reply:
Updated.